com.endlessm.CompanionAppService.json.in

```diff
@@ -16,6 +16,7 @@
         "--filesystem=~/.config/dconf:ro",
         "--filesystem=/var/lib/flatpak:ro",
         "--filesystem=/var/endless-extra/flatpak:ro",
+        "--filesystem=host:ro",
```

Comment:
Would be nice if we could only mount `/usr/share/eos-companion-app` and `/etc/eos-companion-app` instead of the whole host FS... but no worries if that does not work.

Reply:
Yeah, I am afraid that is not possible. Should file an upstream flatpak ticket.